horace_core/spaghetti_plot.m

```diff
@@ -139,7 +144,7 @@
 end
 
 if numel(args) ~= 2
-    error('SPAGHETTI_PLOT:invalid_arguments', ...
+    error('SPAGHETTI_PLOT:invalid_argument', ...
           'Invalid number of arguments')
 end
```

Comment:
I believe, we have agreement to throw 3-component exceptions. Standard exception would be useful if you, e.g. write GUI and catch all normal exceptions to display them in different place in GUI. 

I do not mind if it is not the case here, but having standard 3-component exception 'HORACE:spaghetti_plot:invalid_argument' would simplify further development

Reply:
Agreed, fixed.